crypto/evp/pmeth_lib.c

```diff
@@ -986,6 +986,21 @@ void EVP_PKEY_meth_set_ctrl(EVP_PKEY_METHOD *pmeth,
     pmeth->ctrl_str = ctrl_str;
 }
 
+void EVP_PKEY_meth_set_digestsign(EVP_PKEY_METHOD *pmeth,
+    int (*digestsign) (EVP_MD_CTX *ctx, unsigned char *sig, size_t *siglen,
+                       const unsigned char *tbs, size_t tbslen))
```

Comment:
It's somewhat interesting that we don't have typedefs for these functions, but it would be weird to add typedefs for just the newly added functions, and I don't want to propose adding typdefs for everything since we don't like these APIs anyway.

Reply:
We do it all over the place for the libcrypto\<-\>provider interface 😉